src/main/java/org/ambraproject/wombat/service/PeerReviewServiceImpl.java

```diff
@@ -134,11 +144,34 @@ String getAllReviewsAsXml(List<Map<String, ?>> reviewLetterItems) throws IOExcep
     // append the acceptance letter
     peerReviewContent += acceptanceLetter;
 
+    peerReviewContent += "<received-date>" + articleMetadata.get("receivedDate") + "</received-date>";
```

Comment:
Re: dev note 1.

I would name the tag as close as we can to the corresponding tag in the article XML, for easier troubleshooting.  `article-received-date` is more precise than `received-date` but I don't feel strongly about it.  Your call.

Reply:
renamed to article-received-date.